src/components/Sidebar.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useLocation } from "react-router-dom";
+import { sidebarData } from "toolkit/data/sidebarData";
+import { useSidebar } from "contexts/SidebarContext";
+
+export const Sidebar = () => {
+  const { showFilterBar, setShowFilterBar } = useSidebar();
+  const location = useLocation();
+  return (
+    <div className={`side-bar video-sidebar w-20p ${showFilterBar ? "show-filter" : ""}`}>
+      <ul className="sidebar-items no-bullet col-flex flex-start no-wrap p-h-2 m-v-3">
+        {sidebarData?.map(({ _id, title, link, icon }) => {
+          const Icon = icon;
```

Comment:
suggestion: you can use icon: Icon in the destructing itself

Reply:
Done with change.